tests/utils.py

```diff
@@ -18,7 +19,11 @@ def make_request(data, method='POST'):
     """ Make a webob JSON Request """
     request = Request.blank('/')
     request.method = 'POST'
-    request.body = json.dumps(data).encode('utf-8') if data is not None else ""
+    data = json.dumps(data).encode('utf-8') if data is not None else ""
```

Comment:
Why not just make the else clause like `else b""`  which will provide bytes version of json string or an empty bytestring?